Lecture4/webservices/webservices/ConstantsForMethods.java

```diff
@@ -0,0 +1,18 @@
+package webservices;
+
+public enum ConstantsForMethods {
+
+    REGISTER_USER("https://taschool-notes-service.herokuapp.com/v1/accounts"),
```

Comment:
Imho it's really cool idea to keep a resources in the separate place, but I'd like to suggest to not hardcode the host of the service